Apply Transform Over Each Element In Array/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @param {number[]} arr
+ * @param {Function} fn
+ * @return {number[]}
+ */
+var map = function(arr, fn) {
+    return arr.map((e, i)=>fn(e, i));
```

Comment:
```suggestion
    return arr.map((item, index)=>fn(item, index));
```
Great, however always try to give meaning full name to the variables as best practice.